prime-angular-frontend/src/app/modules/phsa-labtech/shared/services/phsa-labtech-form-state.service.ts

```diff
@@ -28,21 +29,23 @@ export class PhsaFormStateService extends AbstractFormStateService<PhsaEnrollee>
   }
 
   public get json(): PhsaEnrollee {
-    const { phone, phoneExtension, email } = this.demographicsForm.getRawValue();
-    // TODO: Needs to get value (code) of selected checkboxes
-    const { partyTypes } = this.availableAccessForm.getRawValue();
+    // const accessCode = this.accessCodeFormState.json;
```

Comment:
should this be here?

Reply:
I don't know.  Was removed when I was on vacation